myosuite/envs/myo/walk_v0.py

```diff
@@ -6,8 +6,10 @@
 import collections
 import gym
 import numpy as np
+import mujoco_py
 
 from myosuite.envs.myo.base_v0 import BaseV0
+from scipy.spatial.transform import Rotation as R
```

Comment:
@P-Schumacher  -- Do we need this dependency? Can we use the [myosuite_utils](https://github.com/facebookresearch/myosuite/blob/main/myosuite/utils/quat_math.py)?